composer.json

```diff
@@ -26,7 +26,10 @@
     "require-dev": {
         "friendsofphp/php-cs-fixer": "^2.14",
         "phpunit/phpunit": "^8",
-        "symfony/http-client": "^4.3|^5.0"
+        "symfony/http-client": "^4.3|^5.0",
+        "symfony/messenger": "^4.4|^5.0",
+        "symfony/framework-bundle": "^4.3|^5.0",
+        "symfony/http-foundation": "^4.3|^5.0"
```

Comment:
1. Could we sort packages by name?
2. and add the composer config option to automatically sort packages